app/src/main/java/app/gamenative/ui/widget/PerformanceHudView.kt

```
@@ -0,0 +1,243 @@
+
+        return ((busy * 100L) / total).toInt().coerceIn(0, 100)
+    }
+
+    private fun readUsedRamText(): String {
+        val activityManager = context.getSystemService(Context.ACTIVITY_SERVICE) as? ActivityManager ?: return "—"
+        val info = ActivityManager.MemoryInfo()
+        activityManager.getMemoryInfo(info)
+        val usedBytes = (info.totalMem - info.availMem).coerceAtLeast(0L)
+        val usedGb = usedBytes.toDouble() / (1024.0 * 1024.0 * 1024.0)
+        return if (usedGb >= 1.0) {
+            String.format("%.1fGB", usedGb)
```

detekt (1.23.8)

[warning] String.format("%.1fGB", usedGb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)